themes/openy_themes/openy_carnation/openy_carnation.theme

```diff
@@ -495,6 +495,15 @@ function openy_carnation_theme_suggestions_form_alter(array &$suggestions, array
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function openy_carnation_theme_suggestions_views_view_unformatted_alter(array &$suggestions, array $variables) {
+  if ($variables['view']->id() == 'taxonomy_term' && isset($variables['rows'][0]['#node']) && $variables['rows'][0]['#node'] instanceof NodeInterface) {
```

Comment:
Can we use the early return pattern here to simplify the condition?

Reply:
Something like that looks more readable to me
```
if ($variables['view']->id() != 'taxonomy_term') {
  return;
}
if (isset($variables['rows'][0]['#node']) && $variables['rows'][0]['#node'] instanceof NodeInterface) {
  $suggestions[] = 'views_view_unformatted__' . $variables['view']->id() . '__' . $variables['rows'][0]['#node']->getType();
}
```

Reply:
Fixed

Reply:
https://github.com/ymcatwincities/openy/wiki/Code-Review-Quality-Best-Practices#return-early-pattern

Reply:
or 
```
if ($variables['view']->id() != 'taxonomy_term' || !isset($variables['rows'][0]['#node']) ) {
  return;
}
if ($variables['rows'][0]['#node'] instanceof NodeInterface) {
  $suggestions[] = 'views_view_unformatted__' . $variables['view']->id() . '__' . $variables['rows'][0]['#node']->getType();
}
```

Reply:
This is what vacation does for good devs :D

Reply:
:rofl: